### ISSUE TITLE
Error with grids trying to create_newcase for ne1024 after upstream merge

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/scream#1207]
When trying to run a "usual" ne1024 v0.1 job, I hit the following:

```
thetalogin5% create_newcase --case c --compset F2010-SCREAM-HR-DYAMOND2 --res ne1024pg2_r0125_oRRS18to6v3
Compset longname is 2010_EAM%SCREAM-HR-DYAMOND2_ELM45%SPBC_CICE%PRES_DOCN%DOM_MOSART_SGLC_SWAV
Compset specification file is /lus/theta-fs0/projects/ClimateEnergy_4/ndkeen/wacmy/E3SM_code/beydoun-no_p3_sbgr_enhancement-sep17/components/eam//cime_config/config_compsets.xml
Automatically adding SIAC to compset
Automatically adding SESP to compset
ERROR: Unexpected number of colon-delimited elements in ATM2ROF_FMAPNAME: cpl/gridmaps/ne1024pg2/map_ne1024pg2_to_r0125_mono.200212.nc:cpl/gridmaps/ne1024pg2/map_ne1024pg2_to_r0125_mono.200212.nc (expected 1 elements)
```
(same thing on cori)

I then commented out what looks like offending section in `cime_config/config_grids.xml`:
```
-    <gridmap atm_grid="ne1024np4.pg2" rof_grid="r0125">
+    <!-- ndk gridmap atm_grid="ne1024np4.pg2" rof_grid="r0125">
       <map name="ATM2ROF_FMAPNAME">cpl/gridmaps/ne1024pg2/map_ne1024pg2_to_r0125_mono.200212.nc</map>
       <map name="ATM2ROF_SMAPNAME">cpl/gridmaps/ne1024pg2/map_ne1024pg2_to_r0125_mono.200212.nc</map>
-    </gridmap>
+    </gridmap-->
```

which allowed the case to run.

**Comment by PeterCaldwell** [comment]
I think we don't really care about river runoff for our simulations since no-interactive-ocean means ROF doesn't have any impact on our solution - it just gives us extra output variables. That said, perhaps @wlin7 knows if water cycle changed the way ROF needs to be specified?

**Comment by wlin7** [comment]
@PeterCaldwell , @ndkeen , water cycle indeed changes to use MOSART (in place of SROF) for river.

Thanks for isolating the issue, Noel. I ran into that problem when testing MPASSI for F-case with master. I didn't figure out why so I went ahead with an older version of repo to test F-MPASSI.

I think I know what happened. We had a PR for bi-grid ne1024pg2 merged to SCREAM first. There is a similar PR for E3SM focusing on land. The cited block of text were not inserted at the same place inb config_grids.xml. SCREAM and E3SM would work fine previously. After the upstream merge, it became a duplication. I am going to put in a PR to remove the duplication in SCREAM (and make sure the final location is as in E3SM).
